serving/docs/model.md

```diff
@@ -0,0 +1,384 @@
+# DJL Serving Model Packaging
+
+## Overview
+
+DJL Serving is a high-performance serving system for deep learning models. DJL Serving supports models with:
+
+1. [Python Mode](#python-mode)
+2. [Java Mode](#java-mode)
+3. [Binary Mode](#binary-mode)
+
+In addition to the mode specific files, the `serving.properties` is a configuration file that can be used in all modes. Place `serving.properties` in the same directory with your model file to specify configuration for each model.
+
+In `serving.properties`, you can set number of workers for each model, or you can configure minimum workers and maximum workers differently for GPU and CPU.
```

Comment:
Note: loading model in Python mode is pretty heavy. We recommend to set `mxinWorker` and `maxWorker` to be the same value to avoid unnecessary load and unload.